src/main/java/org/cbio/gdcpipeline/processor/ClinicalDataProcessor.java

```diff
@@ -0,0 +1,41 @@
+package org.cbio.gdcpipeline.processor;
+
+import org.cbio.gdcpipeline.model.cbio.CBioClinicalDataModel;
+import org.cbio.gdcpipeline.model.cbio.SampleFileModel;
+import org.springframework.batch.item.ItemProcessor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ClinicalDataProcessor implements ItemProcessor<CBioClinicalDataModel, CBioClinicalDataModel> {
+    @Override
+    public CBioClinicalDataModel process(CBioClinicalDataModel patient) throws Exception {
+
+        //CBioClinicalDataModel modified = modifySampleList(patient);
+
+        //return modified;
+
+        return patient;
+
+    }
+
+    protected CBioClinicalDataModel modifySampleList(CBioClinicalDataModel patient) {
```

Comment:
If we know we are going to filter out normal samples (from the job parameters specified by the user) we could probably do the filtering step before the processor.